src/options.rs

```diff
@@ -30,6 +30,12 @@ pub struct Options {
     help = "Set index cache to <DB_CACHE_SIZE> bytes. By default takes 1/4 of available RAM."
   )]
   pub(crate) db_cache_size: Option<usize>,
+  #[arg(
+    long,
+    default_value = "5000",
+    help = "Commit index every <COMMIT_INTERVAL> blocks."
```

Comment:
Sorry, my mistake, this should be:

```suggestion
    help = "Commit to index every <COMMIT_INTERVAL> blocks."
```